test/ig/IGExporter.IG.test.ts

```diff
@@ -0,0 +1,548 @@
+import fs from 'fs-extra';
+import path from 'path';
+import temp from 'temp';
+import { IGExporter } from '../../src/ig';
+import { StructureDefinition, InstanceDefinition, CodeSystem } from '../../src/fhirtypes';
+import { Package } from '../../src/export';
+import { Configuration, PackageJSON } from '../../src/fshtypes';
+import { FHIRDefinitions, loadFromPath } from '../../src/fhirdefs';
+import { loggerSpy } from '../testhelpers';
+
+describe('IGExporter', () => {
+  temp.track();
+
+  describe('#simple-ig', () => {
+    let pkg: Package;
+    let exporter: IGExporter;
+    let tempOut: string;
+
+    beforeAll(() => {
+      const defs = new FHIRDefinitions();
+      loadFromPath(
+        path.join(__dirname, '..', 'testhelpers', 'testdefs', 'package'),
+        'testPackage',
+        defs
+      );
+      const fixtures = path.join(__dirname, 'fixtures', 'simple-ig');
+      // several parts of the IG exporter still need packageJSON to function
+      const packageJSON: PackageJSON = fs.readJSONSync(path.join(fixtures, 'package.json'));
+      const config: Configuration = {
+        filePath: path.join(fixtures, 'config.yml'),
+        id: 'sushi-test',
+        url: 'http://hl7.org/fhir/sushi-test',
+        version: '0.1.0',
+        name: 'sushi-test',
+        title: 'FSH Test IG',
+        description: 'Provides a simple example of how FSH can be used to create an IG',
+        dependencies: [
+          { packageId: 'hl7.fhir.r4.core', version: '4.0.1' },
+          { packageId: 'hl7.fhir.us.core', version: '3.1.0' },
+          { packageId: 'hl7.fhir.uv.vhdir', version: 'current' }
+        ],
+        status: null,
```

Comment:
Since I think we shouldn't default status anymore, and instead we should require status in the config, I think maybe we should just set status to a real value here.